common/styles/appfooter.css

```diff
@@ -0,0 +1,12 @@
+footer .container1 {
```

Comment:
I can't seem to understand why we have extra ones. As I mentioned early to @RFSH, I think `appheader` has to be separate because of the way the nav bar works but I could be wrong. Part of me thinks that there shouldn't be 3 separate css documents as well.